src/lib/Tests/Form/DataTransformer/ContentInfoTransformerTest.php

```diff
@@ -48,10 +52,8 @@ public function testTransform($value, $expected)
 
     /**
      * @dataProvider transformWithInvalidInputDataProvider
-     *
-     * @param $value
      */
-    public function testTransformWithInvalidInput($value)
+    public function testTransformWithInvalidInput($value): void
```

Comment:
missing strict type

Reply:
It's `mixed` on propose. See test cases in the related data provider.

Reply:
> On purpose there is no type spec for `$value` parameter. See test cases in the related data provider.

Ok, so I see now that we're testing something which should be guaranteed on the language level.
In this particular case, due to Symfony architecture, I would allow `@param mixed $value`. `\Symfony\Component\Form\DataTransformerInterface::transform` is documented like that anyway.